### PR TITLE
Add section and images for adding internal and external testers

### PR DESCRIPTION
**Pull request by SoaringEarth**

### CODE REVIEW

File: operations/itunestesters/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The process to add a user as an internal tester is as follows:
 
  1. After logging in, you will be presented with the following screen, **select 'users and access'**
  <p align="center">
-     <img src="/internal/itunesConnect_homepage.png">
+     <img src="Internal/itunesConnect_homepage.png">
```
Comment (rossrossp) [inline operations/itunestesters/README.md]: How come the URL change?

Comment (SoaringEarth) [reply operations/itunestesters/README.md]: The folder had a capital, you can't rename folders online, and the repo isnt tracking folder name changes :(

**Review by mickeylee25 — APPROVED** [review]

File: operations/itunestesters/README.md (hunk 1 above)
Comment (thisolivier) [reply operations/itunestesters/README.md]: So technically you can, you just have to move the files to a new directory (delete the old one), commit that, then move them back and commit again.

File: operations/itunestesters/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Adding Testers
+
+## Internal Testers/Users
+
+### What is an internal tester?
+An internal tester is a user that is usually part of the host accounts organisation, someone that will need to access any resources that iTunesConnect has. An internal tester/user will be someone that needs to make use of the iTunesConnect website in ways such as setting up the app store page, updating testers, filling in billing information or viewing analytics data.
+
+### Who should be an internal tester?
+Someone should be an internal tester if they require access to any and all builds that are submitted to the host account, **this can include builds that contain issues or may be broken** so think carefully about whether a particular user needs to receive every build or can they simply be added as an external tester which we will cover in the section at the bottom of this guide.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: Syntax change, break long sentence into two:

> ... added to every **build. If not, they** can simply be added as an external tester**,** which we will cover in the section at the bottom of this guide.

File: operations/itunestesters/README.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+ <p align="center">
+     <img src="internal/internalUser_list.png">
+ </p>
+
+3. **Select the blue '+' icon** and you will be presented with a dialogue box as similar to the one below
+ <p align="center">
+     <img src="internal/internalUser_dialogueBox.png">
+ </p>
+
+4. Fill in the information required and set a permissions level, depending on how much access you wish to give the user. Admin affords the greatest level of permissions and is capable of performing all tasks, if the new user is required to add and configure new and existing users then "App Manager" should be sufficient, if the user is being added only to receive internal builds and nothing else, then "Marketing" or "Customer Support" would be fine.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: Sentence breaks rather than commas.

File: operations/itunestesters/README.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+<p align="center">
+    <img src="internal/internalUser_dialogueBox_complete.png">
+</p>
+
+5. After **tapping the 'invite' button** you should be directed back to the list of internal testers with a new user added, they will have received an email inviting them to the account, **they must accept this** otherwise they will not be able to be added to apps as testers and subsequently not receive any builds of apps that are released.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: > ...with a new user added. The new user will...

File: operations/itunestesters/README.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+<p align="center">
+    <img src="internal/internalTester_build_dialogue.png">
+</p>
+
+9. Once the user has downloaded the testFlight app and tapped the link in the email, they will then be able to access all existing and future builds of the app.
+
+## External Testers
+
+### **NOTE**
+External testers are added to external builds of the app, an external build is a specific build that needs to be sent for 'beta approval', this is a miniature review that apple does to ensure the app is stable and safe for use by testers as any build distributed this way be sent to up to 10,000 users.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: > builds of the app. An external

File: operations/itunestesters/README.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+**The beta review process can take between 24-48 hours** and will only be available to external testers if it successfully passes this process.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: > **Apple's beta review process can take between 24-48 hours**

File: operations/itunestesters/README.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+Another thing to mention is that only the first build of a version requires external review, subsequent builds of the same version will be automatically approved.
+
+For example:
+You submit version 11.1 build 11234 for external review, after 24 hours, the build is approved and your testers are happy finding bugs and raising issues. Your friendly neighbourhood developers lovingly resolve the issues and another build is uploaded to iTunesConnect, version 11.1 build 11235, as they share the same version number, build 11235 can be submitted for external review again and will automatically be approved.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: Make it numbered list for clarity (optional)

File: operations/itunestesters/README.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+If there were substantial changes that required a version change, upping the version to 11.2 for example, this would then require the full external review process of 24-48 hours.
+
+### What is an external tester?
+An external tester is someone who will only want to receive select builds, either they do not need to see every build, or they are added to test at specific times, such as UAT (User Acceptance Testing) sessions or when requiring sign-off.
+
+### Who should be an external tester?
+A user should be added as an external tester if they are likely to report issues and distract from the app development process, in this case, it is best the user is added as an external so they see the best, curated builds that are in a better position to be demo'ed.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: > A user should be added as an external tester if they are likely to report issues and distract from the app development process, in this case, it is best

Not sure this makes sense. I think the intention is to say something like
"Some testers may give too much feedback and not understand that many development builds contain known bugs. By adding them as an external tester, you can control which builds they are asked to test, ensuring they only see curated builds that are in a better position to be demo'ed."

File: operations/itunestesters/README.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+<p align="center">
+    <img src="external/itunesConnect_homepage.png">
+</p>
+
+2. Navigate in to the app you wish to add testers to, **select 'testflight' from the top bar** and you will be presented with the following page
+<p align="center">
+    <img src="external/testflight_builds_page.png">
+</p>
+
+3. **Select 'New Group'** from the left side menu, entering a group name in the dialogue box that appears, the group will be where you can put different users, different groups can be added to the same or different builds.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: > users. Different groups

File: operations/itunestesters/README.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+10. The next step involves adding some testing information, this can range from a few words to full paragraphs about what you want the users to look out for or test in this particular version.
+<p align="center">
+    <img src="external/build_submission_test_info.png">
+</p>
+<p align="center">
+    <img src="external/build_submission_test_info_filled.png">
+</p>
+
+11. After continuing from the test information dialogue, you will be redirected back to your list of builds where the selected build is now queued for review, testFlight notifications will be emailed to users within the added group once it has been approved for external testing.
```
Comment (thisolivier) [inline operations/itunestesters/README.md]: > ...review. testFlight notifications...

**Review by thisolivier — CHANGES_REQUESTED** [review]
Syntax changes

File: operations/itunestesters/README.md (hunk 1 above)
Comment (SoaringEarth) [reply operations/itunestesters/README.md]: this has been done, change is outdated

**Comment by SoaringEarth** [comment]
@thisolivier @rossrossp @mickeylee25 Fixed all feedback

**Review by thisolivier — APPROVED** [review]
Nice nice NICE

**Pull request merged**